__tests__/app.test.js

```diff
@@ -142,8 +147,100 @@ describe("/api/articles/:article_id", () => {
 	});
 });
 
+describe("/api/articles/:article_id/comments", () => {
+	describe("GET: 200", () => {
+		it("should return an array of comments for the given article", () => {
+			const article_id = 1;
+			return request(app)
+				.get(`/api/articles/${article_id}/comments`)
+				.expect(200)
+				.then(({ body }) => {
+					const { comments } = body;
+					expect(comments).toHaveLength(11);
+					comments.forEach((comment) => {
+						expect(comment).toHaveProperty(
+							"comment_id",
+							expect.any(Number)
+						);
+						expect(comment).toHaveProperty(
+							"votes",
+							expect.any(Number)
+						);
+						expect(comment).toHaveProperty(
+							"created_at",
+							expect.any(String)
+						);
+						expect(comment).toHaveProperty(
+							"author",
+							expect.any(String)
+						);
+						expect(comment).toHaveProperty(
+							"body",
+							expect.any(String)
+						);
+						expect(comment).toHaveProperty(
+							"article_id",
+							expect.any(Number)
+						);
+					});
+				});
+		});
+		it("should return comments most recent first", () => {
+			const article_id = 1;
+			return request(app)
+				.get(`/api/articles/${article_id}/comments`)
+				.expect(200)
+				.then(({ body }) => {
+					const { comments } = body;
+					expect(comments).toBeSortedBy("created_at", {
+						descending: true,
+					});
+				});
+		});
+	});
+
+	describe("GET: 404", () => {
```

Comment:
be careful with your nested describes here, the describe block is suggesting that you are testing for 404s but your tests inside test for 404, 200 and 400s